.github/workflows/tests.yml

```diff
@@ -32,9 +32,6 @@ jobs:
             laravel: 9.*
             framework: ^9.0
             stability: prefer-stable
-        exclude:
-          - laravel: 9.*
-            php: 7.4
```

Comment:
Please revert these changes. We need to continue testing PHP 7.4.

Reply:
Ah, bummer. It was a bit sad to see all those PHP 8 features get removed in that PR. I'm sure everyone who needs to use 7.4 will be appreciative though! Thanks!